zone/gm_commands/guild.cpp

```diff
@@ -476,9 +502,215 @@ void command_guild(Client *c, const Seperator *sep)
 			}
 		}
 	}
+	else if (is_details)
+	{
+		auto guild_id = Strings::ToUnsignedInt(sep->arg[2]);
+		auto guild = guild_mgr.GetGuildByGuildID(guild_id);
+		if (!guild) {
+			c->Message(Chat::Yellow, fmt::format("Guild {} not found.  #guild list can be used to get guild ids.", guild_id).c_str());
+			return;
+		}
+
+		if (guild) {
+			c->Message(Chat::Yellow, fmt::format("Guild ID:         {}.", sep->arg[2]).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild Name:       {}.", guild->name.c_str()).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild Leader ID:  {}.", guild->leader).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild MinStatus:  {}.", guild->minstatus).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild MOTD:       {}.", guild->motd.c_str()).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild MOTD Setter:{}.", guild->motd_setter.c_str()).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild Channel:    {}.", guild->channel.c_str()).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild URL:        {}.", guild->url.c_str()).c_str());
+
+			for (int i = 1; i <= GUILD_MAX_RANK; i++) {
+				c->Message(Chat::Yellow, fmt::format("Guild Rank:       {} - {}.", i, guild->rank_names[i].c_str()).c_str());
+			}
+
+			c->Message(Chat::Yellow, "Guild Functions:   {db_id} - {guild_id} - {perm_id} - {perm_value}.");
+			for (int i = 1; i <= GUILD_MAX_FUNCTIONS; i++) {
+				c->Message(Chat::Yellow, fmt::format("Guild Function:   {} - {} - {} - {}.",
+					guild->functions[i].id,
+					guild->functions[i].guild_id,
+					guild->functions[i].perm_id,
+					guild->functions[i].perm_value
+				).c_str());
+			}
+			c->Message(Chat::Yellow, fmt::format("Guild Tribute:  Favor     {}", guild->tribute.favor).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild Tribute:  Tribute 1 {}/{}   - Tribute 2 {}/{}",
+				guild->tribute.id_1,
+				guild->tribute.id_1_tier,
+				guild->tribute.id_2,
+				guild->tribute.id_2_tier
+			).c_str());
+			c->Message(Chat::Yellow, fmt::format("Guild Tribute:  Time Remaining {} - Enabled {}",
+				guild->tribute.time_remaining,
+				guild->tribute.enabled
+			).c_str());
+		}
+		for (auto& c1 : entity_list.GetClientList()) {
+			if (c1.second->GuildID() == guild_id) {
+				c->Message(Chat::Yellow, fmt::format("PlayerName: {} ID: {} Rank: {} OptIn: {} DirtyList: {}.",
+					c1.second->GetCleanName(),
+					c1.second->GuildID(),
+					c1.second->GuildRank(),
+					c1.second->GuildTributeOptIn(),
+					c1.second->GetGuildListDirty()
+				).c_str());
+			}
+		}
+	}
+	else if (is_test)
```

Comment:
Is this a real command or just something bogus you were messing with?

Reply:
Seems like bogus code to me just based on it all being test stuff for making sure things are sent properly.

Reply:
It is for my testing as noted in the original notes on the PR.  I added several guild commands for easy testing of functions.  I would want to remove them once things are finalized.  I could do the same for api functions.  I will take your lead.

Reply:
I have reverted to the master guild.cpp in order to remove all the test commands.